scripts/apps/master-desk/components/FilterBarComponent.tsx

```diff
@@ -1,11 +1,13 @@
 import {gettext} from 'core/utils';
+import _ from 'lodash';
 import React from 'react';
 
 import {IFilter} from './FilterPanelComponent';
 
 interface IProps {
     filters: IFilter;
     onFilterChange(filters: any): void;
+    setClearAllFilters(): void;
```

Comment:
It'd be good to limit coupling between filter-bar and other components.

For example, we want to clear both - applied filters and editable filters in the sidebar. To make things less coupled, this component shouldn't have any information about the sidebar filters. Using the following interface would accomplish it:

```
interface IProps {
    filters: IFilter;
    removeFilter(id: string): void;
    removeAllFilters(): void;
}
```